### PR TITLE
announce the move to clang 10

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @isuruf

### CODE REVIEW

File: src/user/announcements.rst [hunk 1]
```diff
@@ -5,6 +5,12 @@ Announcements
 ----
 
 
+:2020-07-15: Moving from clang 9 to clang 10
```
Comment (ocefpaf) [inline src/user/announcements.rst:8]: I'm not sure how ReST works, should we have only a single `2020-07-15` header? If so just offsetting by the same amount of spaces the CFEP-18 would work?

File: src/user/announcements.rst [hunk 2]
```diff
@@ -5,6 +5,12 @@ Announcements
 ----
 
 
+:2020-07-16: Moving from clang 9 to clang 10
+
+    conda-forge is moving to clang 10!
```
Comment (xhochy) [inline src/user/announcements.rst:10]: This is probably only for macOS, we don't use it somewhere else.
```suggestion
    conda-forge is moving to clang 10 on macOS!
```

File: src/user/announcements.rst (hunk 1 above)
Comment (ocefpaf) [reply src/user/announcements.rst]: I like things that solves themselves by just waiting a little bit :smile:

**Comment by ocefpaf** [comment]
@isuruf is this OK to be merge? I'll send the other one for openblas as soon as we get this one in.

**Comment by ocefpaf** [comment]
Alright, the updates were approved. Merging this.

**Pull request merged**